engine/src/main/scala/cromwell/engine/io/gcs/GcsBatchFlow.scala

```diff
@@ -54,7 +54,33 @@ class GcsBatchFlow(batchSize: Int, scheduler: Scheduler, onRetry: IoCommandConte
     }
   }
 
-  private val batchRequest: BatchRequest = {
+  /**
```

Comment:
I don't know what the conventions are for the cromwell code base, but note that this is not valid javadoc. If it was rendered as javadoc, all the text here would be rendered as a single paragraph with no breaks or formatting. Since this isn't a public API, there's no need to make this a javadoc comment.

Reply:
Cromwell doesn't have a strong set of "conventions". 😭

Maybe one day...

Thanks for the heads up though!